### DISCUSSION TITLE
Generate ACTION_URLs

### DISCUSSION THREAD

**Discussion by JustMrMendez** [pocketbase/pocketbase#2517] [Q&A]
Account transactional emails require a smpt server, but the email i went with only has an API, is there a way to either generate the `ACTION_URLs` so i can send the emails myself?

**Comment by ganigeorgiev** [comment]
Currently the email tokens can be generated only programmatically with Go using the methods available in the [`tokens` subpackage](https://pkg.go.dev/github.com/pocketbase/pocketbase@v0.15.3/tokens).
